packages/renderer/src/components/GetOtp.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import styles from './GetOtp.module.css';
+import { Button, FormControl, Modal } from 'react-bootstrap';
+import { observer } from 'mobx-react-lite';
+import { useConfigStore } from '/@/store/ConfigStore';
+import { sendOTPResponse } from '#preload';
+
+const GetOtp = () => {
+  const configStore = useConfigStore();
+  const [modalStatus, setModalStatus] = useState<boolean>(false);
+  const [inputText, setInputText] = useState('');
+
+  const closeModal = () => setModalStatus(false);
```

Comment:
OK, you allow the user to close the modal, but the backend process will still wait for the OTP.

If the user closed the modal, I assume you should `sendOTPResponse(undefined)` or something to fail the process.

Reply:
Done